spikingjelly/datasets/base.py

```
@@ -334,6 +334,35 @@ def build_impl(self) -> None:
     def build(self) -> Tuple[Path, Callable]:
+        r"""
+        **API Language:**
+        :ref:`中文 <EventBuilder.build-cn>` | :ref:`English <EventBuilder.build-en>`
+
+        ----
+
+        .. _EventBuilder.build-cn:
+
+        * **中文**
+
+        直接使用原始数据集目录作为处理后的数据集目录，不做额外处理。
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)